pkgs/development/tools/misc/findnewest/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "findnewest";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "0-wiz-0";
+    repo = pname;
+    rev = name;
```

Oops, I didn't realize the rev was the `name`, which is now no longer available in the attrset passed to mkDerivation. So I guess:

```suggestion
    rev = "${pname}-${version}";
```